src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -0,0 +1,118 @@
+import path from 'path'
+
+import { FormAuthType } from '../../../../shared/types'
+import config from '../../config/config'
+import { ControllerHandler } from '../core/core.types'
+import * as FormService from '../form/form.service'
+import * as PublicFormController from '../form/public-form/public-form.controller'
+import { RedirectParams } from '../form/public-form/public-form.types'
+import * as HomeController from '../home/home.controller'
+
+const RESPONDENT_COOKIE_NAME = 'v2-respondent-ui'
+const ADMIN_COOKIE_NAME = 'v2-admin-ui'
+
+export type SetEnvironmentParams = {
+  ui: 'react' | 'angular'
+}
+
+export const RESPONDENT_COOKIE_OPTIONS = {
+  httpOnly: true,
+  sameSite: 'strict' as const,
+  secure: !config.isDev,
+}
+
+export const ADMIN_COOKIE_OPTIONS = {
+  httpOnly: true,
+  maxAge: 365 * 24 * 60 * 60, // 1 year
+  sameSite: 'strict' as const,
+  secure: !config.isDev,
+}
+
+const serveFormReact: ControllerHandler = (_req, res) => {
+  const reactFrontendPath = path.resolve('dist/frontend')
+  res.sendFile(path.join(reactFrontendPath, 'index.html'))
```

Comment:
Add return before all `res.whatever()`

Reply:
#bb1239ef545269503c02ee7ff2caa988492e6b16

Just to note: adding the returns is to ensure we don't have accidental multiple data sending on `res`, because that can cause runtime exceptions.

In this file, it feels a little unnecessary because the code is structured in such a way that it should never happen. All the functions which return data do so at the very end, and even in the case of conditionals, the conditional evaluation is itself done at the very end, which makes it clear nothing should go after it.

```javascript
if (condition) {
    res.send(/*... */)
} else {
    res.send(/*... */)
}
```